<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GGMainWindow</class>
 <widget class="QMainWindow" name="GGMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="GGEditorView" name="scEditView">
       <property name="dragMode">
        <enum>QGraphicsView::RubberBandDrag</enum>
       </property>
      </widget>
      <widget class="QStackedWidget" name="stkDetailEdits">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pageEmpty">
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>40</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>EMPTY</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pageStart">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>80</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>START</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pageEnd">
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>110</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>END</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pageCondition">
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>30</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>CONDITION</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pageAction">
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>20</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>ACTION</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pageDecision">
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>40</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>DECISION</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pageLink">
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>40</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>LINK</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="separator"/>
   <addaction name="actionP"/>
   <addaction name="actionS"/>
   <addaction name="actionE"/>
   <addaction name="actionC"/>
   <addaction name="actionA"/>
   <addaction name="actionD"/>
   <addaction name="actionL"/>
  </widget>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S</string>
   </property>
   <property name="toolTip">
    <string>Create Start Page</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>E</string>
   </property>
   <property name="toolTip">
    <string>Create End Page</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="toolTip">
    <string>Create Condition Page</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="toolTip">
    <string>Create Action Page</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>D</string>
   </property>
   <property name="toolTip">
    <string>Create Decision Page</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
  </action>
  <action name="actionL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>L</string>
   </property>
   <property name="toolTip">
    <string>Create Connection</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>P</string>
   </property>
   <property name="toolTip">
    <string>Pointer</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GGEditorView</class>
   <extends>QGraphicsView</extends>
   <header>view/ggeditorview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_New</sender>
   <signal>triggered()</signal>
   <receiver>GGMainWindow</receiver>
   <slot>newModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>GGMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectPage(GGPage*)</slot>
  <slot>selecConnection(GGConnection*)</slot>
  <slot>newModel()</slot>
  <slot>setClickMode()</slot>
 </slots>
</ui>
